ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/FileSystemHelperImpl.java

```diff
@@ -90,8 +108,13 @@ public ObjectStorageSpaceVerificationResult objectsFromBucketWillFitInDirectory(
                     requiredSpace, availableSpace, ioException);
         }
 
-        if ( ! pathIsWritable(destinationDirectory)) {
-            final IOException ioException = null;
+        try {
+            if ( ! pathIsWritable(destinationDirectory)) {
+                final IOException ioException = null;
+                return new ObjectStorageSpaceVerificationResult(ObjectStorageSpaceVerificationResult.VerificationStatus.PathLacksAccess,
+                        requiredSpace, availableSpace, ioException);
```

Comment:
Is there a reason why you use `ioException = null` instead of just passing null here?